### DISCUSSION TITLE
Setting up with v10 on Umbraco Cloud - a little help needed

### DISCUSSION THREAD

**Discussion by greystate** [vendrhub/vendr-documentation#109] [Q&A] [answered]
Hi there 👋

We finally have a clear case for trying out Vendr for one of our clients and I've started a project on Umbraco Cloud, cloned it down, added the `Vendr` + `Vendr.Deploy` packages; that all seems to work fine - but trying to run the site locally on my Mac requires the SQLite package as well, so I found the docs here: https://vendr.net/docs/core/3.0.0/how-to-guides/configuring-sqlite-support/

My problem is that I have absolutely no idea where the lines referenced here goes:

https://github.com/vendrhub/vendr-documentation/blob/e89d401f8f6bec827bc79ea069eda8fca66ccebb/content/core/3.0.0/how-to-guides/configuring-sqlite-support.md?plain=1#L23-L32

My best guess is in the Startup.cs or Program.cs file - but would be nice to know for sure 😬 

(Or maybe it's partly because the link on `IVendrBuilder` doesn't resolve correctly?)
 
Would love to add the necessary info to the docs for anyone else, once I've figured everything out too 👍 

Thanks!

**Comment by greystate** [comment, marked as answer]
Okay - I think I'm on to something here:

>  (Or maybe it's partly because the link on IVendrBuilder doesn't resolve correctly?)

Looks like it's supposed to go here: https://vendr.net/docs/core/3.0.0/key-concepts/dependency-injection/#registering-dependencies

**Reply by mattbrailsford** [reply]
Yup, that's the one. It goes in `Startup.cs` in the `ConfigureServices` after the `.AddComposers()`. 

It's a bit hard sometimes to decide how much detail to put in examples, and I must admit, those first "Key Concepts" sections are probably written in sequence requiring you to have read the former before you read the latter. I'll give it some thought as to whether we can make this stuff clearer.
